efficientdet/hparams_config.py

```diff
@@ -177,6 +177,10 @@ def default_detection_configs():
   h.train_scale_min = 0.1
   h.train_scale_max = 2.0
   h.autoaugment_policy = None
+  h.use_augmix = False
```

Comment:
Do people need to change mixture_width/mixture_depth/alpha?
If people often need to change it, then having 3 options are better, but if most time they don't need to change them, then it might be better to just use a single option:  h.augmix_params = (1, 3, -1)   # (alpha, width, depth)
WDYT?